terraform/bitbar-devicepool/compute.tf

```diff
@@ -0,0 +1,43 @@
+resource "google_compute_instance" "vm_instance" {
+  count = "${var.host_count}"
+
+  name         = "bitbar-devicepool-${count.index}"
+  machine_type = "f1-micro"
+
+  metadata {
+    // can take multiple, just separate with "\n"
+    sshKeys = "${var.ssh_user}:${var.ssh_key}"
+  }
+
+  boot_disk {
+    initialize_params {
+      image = "ubuntu-1804-lts"
+    }
+  }
+
+  network_interface {
+    # A default network is created for all GCP projects
+    network       = "${google_compute_network.bitbar-net.name}"
+    access_config = {}
+  }
+}
+
+resource "google_compute_firewall" "devicepool-firewall" {
+  name    = "devicepool-firewall"
+  network = "${google_compute_network.bitbar-net.name}"
+
+  allow {
+    protocol = "icmp"
+  }
+
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+
+  source_ranges = ["0.0.0.0/0"]
```

Comment:
What services will be accessing these hosts?  We might consider setting up vpns between gcp and mdc1/2 and aws but that would be a farther out goal.  In the meantime, if the hosts are only going to be accessed from Mozilla networks, we could limit access to the outgoing nat IP of the Datacenters and/or the offices.

Reply:
It's basically me and bc ssh'ing to the hosts to manage them and change configurations. The service queries taskcluster and start bitbar jobs.

Using the netblocks sounds great. Is there a doc with those? I'll ping oremj also.

Reply:
Updated with netblocks from oremj.